.github/workflows/bintray_upload_action.yml

```diff
@@ -1,22 +1,17 @@
 # This is a basic workflow to help you get started with Actions
 
-name: CI
+name: Bintray Upload
 
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the master branch
 on:
   push:
     branches: [ master ]
-  pull_request:
-    branches: [ master ]
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   gradle:
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
-    runs-on: ${{ matrix.os }}
+    runs-on: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
I think we can use simply `runs-on: ubuntu-latest` as we don't have e.g. darwin or mingw targets to build and publish.

Reply:
sure I'll try, it actually only accidentally slipped into this PR, I'll change it separately